app/move/controllers/assign/base.test.js

```diff
@@ -184,4 +220,54 @@ describe('Assign controllers', function() {
       })
     })
   })
+
+  describe('#setAllocationDetails', function() {
+    const mockReq = {
+      form: {
+        options: {},
+      },
+    }
+    const mockLocals = {
+      move: {},
+    }
+    const next = sinon.stub()
+    context('if showAllocationDetails is true', function() {
+      let req
+      let res
+      beforeEach(function() {
+        req = { ...mockReq }
+        res = { locals: { ...mockLocals } }
+        req.form.options.showAllocationDetails = true
+        controller.setAllocationDetails(req, res, next)
+      })
+      afterEach(function() {
+        next.resetHistory()
```

Comment:
If you assign next as `sinon.stub()` within the `beforeEach` block it will use the global `afterEach` to reset the stub after each test case.